provers/cairo/Cargo.toml

```diff
@@ -1,116 +1,16 @@
-
 [package]
-name = "cairo-platinum-prover"
-rust-version = "1.66"
+name = "cairo-platinum-prover-cli"
 version.workspace = true
 edition.workspace = true
 license.workspace = true
+repository.workspace = true
 
-[lib]
-crate-type = ["cdylib", "rlib"]
+[[bin]]
+name = "cairo-platinum-prover-cli"
```

Comment:
i think we could just use the name without the `-cli`, unless there is some naming conflict with the library but I don't think that will be the case